dev-db/etcd-wrapped/files/etcd.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=etcd%i-wrapped
+
+[Service]
+Type=notify
+Environment=ETCD_NAME=%m
+Environment=ETCD_IMG_USER=root
```

Comment:
This can be changed to etcd when https://github.com/coreos/rkt/issues/2831 is dealt with.